experiments/template_SLTC.py

```diff
@@ -134,6 +147,11 @@ def main():
         label2id[l] = n
         id2label[n] = l
 
+    if 'citation' in data_args.finetuning_task:
+        train_dataset = train_dataset.filter(lambda row: str(id2label[row['label']]).startswith("critical"))
+        eval_dataset = eval_dataset.filter(lambda row: id2label[row['label']].startswith("critical"))
+        predict_dataset = predict_dataset.filter(lambda row: id2label[row['label']].startswith("critical"))
+
```

Comment:
What does this do?

Reply:
Here I filter only critical cases, it makes sure only using cases with labels critical-1 critical-2 critical-3 or critical-4. I guess it is reduntant because we implemented it in the lextreme loading script.

Reply:
Yeah I think it's not necessary anymore here too. I think removing it here is better to keep complexity low.

Reply:
ok